.automation/plugins.yml

```diff
@@ -29,3 +29,9 @@ plugins:
     docUrl: https://github.com/ssc-services/mega-linter-plugin-salt
     pluginUrl: https://raw.githubusercontent.com/ssc-services/mega-linter-plugin-salt/main/mega-linter-plugin-salt/salt.megalinter-descriptor.yml
     author: "[Joachim Grimm](https://github.com/grimmjo)"
+  # document-compose-linter
```

Comment:
Why document-compose and not docker-compose ?

Reply:
Indeed, it didn't bother my eye and didn't catch that

Reply:
happens ^^

Reply:
@wesley-dean if you can make another PR to fix it, it would be nice :)

Reply:
@wesley-dean thanks a lot :)